examples/online-shop/README.md

```diff
@@ -0,0 +1,62 @@
+# Ballerina OracleDB module Example use case
+
+## overview 
+This example demonstrates how to use the ballerina oracledb module to execute, query and call etc. in an oracle database. 
+
+## Implementation
+
+These are HTTP RESTful services that are used to insert, create, and retrieve data of a sample database.
```

Comment:
```suggestion
These are HTTP RESTful services used to insert, create, and retrieve data of a sample database.
```

Reply:
corrected in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/36/commits/fe7671fcf8c033cd93419840a549cb2dd8ad96d1